libsolidity/interface/CompilerStack.cpp

```diff
@@ -456,7 +457,7 @@ string const CompilerStack::lastContractName() const
 
 eth::AssemblyItems const* CompilerStack::assemblyItems(string const& _contractName) const
 {
-	if (m_stackState != CompilationSuccessful)
+	if (m_stackState != CompilationSuccessful && !m_parserErrorRecovery)
```

Comment:
There is no way that the compilation is successful when there are analysis errors. This is a security measure.

Reply:
Should be fixed in 2dbe19036